dbcon/mysql/idb_mysql.h

```diff
@@ -28,6 +28,9 @@ template <class T> bool isnan(T);
 #endif
 #endif
 
+#define INFINIDB_DEBUG
+#define DEBUG_WALK_COND
+
```

Comment:
I think these debugging flags are accidentally checked in.

Reply:
Yep